ui/src/store/features/github/async-thunk/github-api.tsx

```diff
@@ -0,0 +1,76 @@
+import { ComboboxItem, DropdownItem } from '@/components';
+import { githubActions, RootState } from '@/store';
+import { useGithub } from '@/views/mint/github-step/steps/use-github';
+import { Repo } from '@/views/mint/mint.context';

```

Comment:
It seems not correct import things from the view in redux files the direction should be the opposite.